openssl/src/ec.rs

```diff
@@ -527,6 +527,30 @@ impl EcPointRef {
             .map(|_| ())
         }
     }
+
+    /// Checks if point is infinity
+    pub fn is_infinity(&self, group: &EcGroupRef) -> Result<bool, ErrorStack> {
```

Comment:
This is not documented to ever return an error, just 0 or 1.

Reply:
Just wanted to make sure, but I see that it does not correspond to other functions..